tests/e2e/pkg/tester/tester.go

```diff
@@ -141,6 +129,29 @@ func (t *Tester) getKopsInstanceGroups() ([]*api.InstanceGroup, error) {
 	return igs, nil
 }
 
+// k/k tests assume the API Server in the kubeconfig is the IP of the control plane node instead of a Loadbalancer.
+// Some tests try to SSH to the API Server IP which fails for kops as the LB doesn't expose port 22
+func (t *Tester) addKubeBastion() error {
+	cluster, err := t.getKopsCluster()
+	if err != nil {
+		return err
+	}
+	if cluster.Spec.LegacyCloudProvider == "gce" {
+		instances, err := kops.GetInstances("kops", cluster.Name, nil)
+		if err != nil {
+			return err
+		}
+		for _, instance := range instances {
+			for _, role := range instance.Roles {
+				if role == "control-plane" && instance.ExternalIP != "" {
+					t.Env = append(t.Env, fmt.Sprintf("KUBE_SSH_BASTION=%s", instance.ExternalIP))
```

Comment:
For a three-node control plane we would set KUBE_SSH_BASTION three times?

Reply:
Forgot about that, we need just one IP.